internal/commands/app/create.go

```diff
@@ -94,11 +101,13 @@ func (cmd *CommandCreate) Flags() []flags.Flag {
 			},
 		},
 		flags.StringFlag{
-			Value: &cmd.inputs.Template,
+			Value:              &cmd.inputs.Template,
+			NoArgsDefaultValue: &noArgsDefaultValueTemplate,
 			Meta: flags.Meta{
 				Name: flagTemplate,
 				Usage: flags.Usage{
 					Description: "Create your Realm app from an available template",
+					Note:        "Manually specifying a template id must be done with a '='. ie: --template=web.graphql.todo",
```

Comment:
this is the limitation of using `NoArgsDefaultValue`

https://github.com/spf13/pflag/issues/134

In practice, I think a relatively few amount of people would encounter this, but I'm hoping that this note is enough to cover us for the portion of users that know the template id and would like to specify them manually

Reply:
ahh this is in fact a little awkward, but I do agree that users are not as likely to specify a template explicitly. what happens if they provide `--template web.graphql.todo`? do they get prompted to select a template? or does it complain about not knowing what `web.graphql.todo` means?

Reply:
The current behavior is that it ignores `web.graphql.todo` and prompts for a template id.

Reply:
im confused by this as well, why does it ignore `web.graphql.todo`?  by ignore im guessing that means `inputs.Template` is blank still?

Reply:
no the value of `inputs.Template` will be the default no-args value which is `<prompt-templates>`

in its current state,

`realm-cli app create --template web.graphql.todo`

is the same as

`realm-cli app create web.graphql.todo --template`

Reply:
it seems like we don't process any keywords after `create` for the `realm-cli app` command so `web.graphql.todo` doesn't really do anything if you do `realm-cli app create --template web.graphql.todo`

Reply:
alright managed to get to the bottom of this offline, I missed what `NoArgsDefaultValue` was entailing at first.  I think we can ultimately avoid this awkwardness while still achieving the same functionality by tackling [REALMC-9512](https://jira.mongodb.org/browse/REALMC-9512) first.

spoke offline with andrew and he said he's happy to take on that ticket first

Reply:
oh wow, interesting... that is indeed a bit hacky/funky, but it ends up working out okay.